examples/test_alias.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+"""Create a JACK client and test port-alias functionality
+
+This client test the 3 functions related with port-aliases
+
+"""
+
+import jack
+
+print('starting TestAlias client')
+
+client = jack.Client('TestAlias')
+
+if client.status.server_started:
+    print('JACK server was started')
+else:
+    print('JACK server was already running')
+if client.status.name_not_unique:
+    print('unique client name generated:', client.name)
+
+ports = client.get_ports()
+
+print('Testing set_alias() ...')
+i=0
```

Comment:
Classic case for `enumerate()`: https://docs.python.org/3/library/functions.html#enumerate.